user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Template_Editor.md

```diff
@@ -336,7 +336,9 @@ To add a condition:
    > For a full overview of all HTML color names, see [HTML Color Names](https://www.w3schools.com/tags/ref_colornames.asp).
 
    > [!NOTE]
-   > Any visual changes made when configuring a conditional case will not show up in the preview.
+   >
+   > - Any visual changes made when configuring a conditional case will not show up in the preview.
+   > - From DataMiner 10.3.0 [CU12]/10.4.3 onwards<!--RN 38368-->, any visual changes made when configuring a conditional case can be reset by clicking the ![reset](~/user-guide/images/Reset_Button.png) button to the right of the option you want to reset. Prior to DataMiner 10.3.0 [CU12]/10.4.3, you can only reset custom text entered for a *Text* layer.
```

Comment:
@TVincke98, thanks for the feedback! I have made a small adjustment based on your feedback. Is this OK to merge now?